bibliopixel/animation/remote/server.py

```diff
@@ -23,6 +23,15 @@ def __init__(self, q_send, q_recv):
         self.q_send = q_send
         self.q_recv = q_recv
 
+    @classmethod
+    def shutdown_server(cls):
+        log.warning('Shutting down UI server.')
+        func = flask.request.environ.get('werkzeug.server.shutdown')
+        if func is None:
```

Comment:
Tiny quibble if you cared:  you could rewrite this as

    return func and (func() or True)
     
or clearer but longer:

    if func:
        func()
        return True

...or... not.  :-D